main.py

```diff
@@ -1,10 +1,44 @@
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
+import json
```

Comment:
Change the order of your imports according to `pep8`. Here `json` should be first.